src/AspNetCore/Src/managedexports.cxx

```diff
@@ -110,17 +117,18 @@ http_set_managed_context(
 }
 
 EXTERN_C __MIDL_DECLSPEC_DLLEXPORT
-VOID
+HRESULT
```

Comment:
what's the motivation for this change?

Reply:
I wanted to standardize all pInvokes to return HRESULTs. In the future, if the implementation of the functions changes and suddenly we can return an invalid HRESULT, I don't want to have to make a second method.